cpp/tests/bitmask/count_bitmask_tests.cu

```diff
@@ -171,3 +173,88 @@ TEST_F(CountUnsetBitsTest, MultipleWordsSingleBit) {
   thrust::device_vector<cudf::bitmask_type> mask(10, cudf::bitmask_type{0});
   EXPECT_EQ(1, cudf::count_unset_bits(mask.data().get(), 67, 68));
 }
+
+struct CopyBitmaskTest
```

Comment:
These tests should go in their own file, or this file should be renamed to `bitmask_tests.cu`.